tox.ini

```diff
@@ -41,17 +41,55 @@ deps = {[testenv]deps}
        gmpy
 setenv = PARAM_TEST_GMPY = 1
 
+# xlrd is the reader for xls files in pandas. xlrd is also the reader
+# for xlsx and xlsm files for pandas<1.2, but this is only possible using
+# xlrd<2. If pandas>=1.2 is guaranteed, you can remove the version spec
```

Comment:
I'd be ok with testing this new file-reading functionality only on pandas>=1.2; it's a new feature and not supporting it with old versions wouldn't be a big deal.

Reply:
Can do, but I think this'll nix python 3.6? I think they only go up to 1.1?

Reply:
I'm ok with not supporting Python 3.6, since we are trying to merge this into Param 2, our forward-looking codebase.